core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -264,15 +300,18 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                 </div>
               </div>
             </div>
-            <button class="btn open-in-editor-btn"
-                    ng-click="showExplorationEditor(exploration.id); $event.stopPropagation()">
-              Open In Editor
-            </button>
+            <a ng-href="/create/<[exploration.id]>">
+              <button class="btn open-in-editor-btn">
+                Open In Editor
+              </button>
+            </a>
           </md-card>
           <div ng-attr-section="'<['right-section']>">
-            <div ng-if="exploration.status === 'private'" class="exp-private-text">
-              <[publishText]>
-            </div>
+            <a ng-href="/create/<[exploration.id]>" style="text-decoration: none;">
```

Comment:
Hmm.. this is kind of buggy. What if the user clicks outside this text and still inside the white section ? It doesn't behave as a link in that case!